### DISCUSSION TITLE
National Adoption Playbook ?

### DISCUSSION THREAD

**Discussion by parthlawate** [Sunbird-Ed/Community#240] [Documentation]
Hello, 

Would there by any documentation that might be available for National level adoption from a perspective of program management, content acquisition, orchestration, scaling etc for national level adoption for Sunbird ED ?

**Comment by jagadishbabu** [comment]
@parthlawate can you please elaborate more on this topic. Are you looking at any specific vertical such as education or agri etc.

**Comment by parthlawate** [comment]
Hello @jagadishbabu Right now i would want to start with Education as there would be experience in the existing teams of orchestrating Diksha..   That could be a starter template which can be created quickly with least knowledge gaps. 

The one I am looking for is for Agri.. But that would be greenfield. So was hoping to derive it from the Edu one.

**Reply by chitranshu-keshav** [reply]
@parthlawate If your issue is resolved can you please update the thread with the details and mark it as answered?

**Reply by parthlawate** [reply]
Dear @chitranshu-keshav -- No its not solved as of now..

**Reply by Krishnaj20** [reply]
@parthlawate - you can reach out to any of the adopters for these details. I'll initiate a mail thread connecting you with adopters in the education domain.

**Reply by parthlawate** [reply]
Thanks. It will be great if we can compile and opensource the knowledge of implementing a national scale education platform. It will help scale adoption to other geographies as well as verticals
